config/default.json

```diff
@@ -75,5 +75,6 @@
   "chain_id": "0000000000000000000000000000000000000000000000000000000000000000",
   "address_prefix": "STM",
   "conveyor_posting_wif": false,
-  "conveyor_username": false
+  "conveyor_username": false,
+  "pinned_posts_url": "https://raw.githubusercontent.com/steemit/condenser-pinned/master/pinned-posts.json"
```

Comment:
We might want this to default to false - thoughts?

Reply:
It'll get overridden by the env variable, but yeah, it's probably safer to not try and show pinned posts at all, if the env variable isn't set. I'll make the change.